### PR TITLE
Second delivery: Add Reducers and Actions

### PR DESCRIPTION
**Pull request by lmunozm1702**
## This PR is for:

> Define action types for adding and removing a book.
- Set the initial state to be an empty array of books.
- Export Action Creators for your actions.
- Write reducers and export it as default.
- Define state changes for the actions created.

> Define an action type for checking the category's status.
- Set the initial state to be an empty array of categories.
- Export Action Creators for actions.
- Write the reducer and export it as default.

> Configure the Redux Store (/src/redux/configureStore.js):
- Import the necessary methods from Redux Toolkit.
- Combine both reducers into a root reducer by using configureStore function.

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (rloterh) [inline src/redux/categories/categories.js]: - [x] Going by the _duck patterns approach_, assigning value to _CHECK_STATUS_ action type should be something like:
```ruby
      CHECK_STATUS = bookstore/categories/CHECK_STATUS;
```
 
Kindly ensure to improve upon this aspect.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Luis, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Good job implementing _duck pattern_ so far!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Live demo link in your ReadMe is broken, kindly endeavor to fix it.
- Kindly endeavor to fill the FAQ section with at least 2 Q&A(s).

Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js (hunk 1 above)
Comment (lmunozm1702) [reply src/redux/categories/categories.js]: Hi @rloterh, it's done, I forget that, thanks!!

**Review by julie-ify — APPROVED** [review]
Hi @lmunozm1702,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**